core/Api/exportCsvApi.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace ICT\Core\Api;
+
+/* * ***************************************************************
+ * Copyright © 2016 ICT Innovations Pakistan All Rights Reserved   *
+ * Developed By: Nasir Iqbal                                       *
+ * Website : http://www.ictinnovations.com/                        *
+ * Mail : nasir@ictinnovations.com                                 *
+ * *************************************************************** */
+
+use ICT\Core\Account;
+use ICT\Core\Api;
+use ICT\Core\CoreException;
+use ICT\Core\User;
+use ICT\Core\User\Permission;
+use SplFileInfo;
+
+class ExportCsvApi extends Api
+{
+
+
+
+  /**
+   * Export Csv
+   *
+   * @url GET /export/users
```

Comment:
Follow project standard for url, note the ending s in users and /csv suffix for file format
    @url GET /users/csv

Reply:
URLs resolved